src/dlls/mscorrc/mscorrc.rc

```diff
@@ -626,7 +626,7 @@ BEGIN
     IDS_EE_BADMARSHAL_NOTMARSHALABLE        "The type definition of this field has layout information but has an invalid managed/unmanaged type combination or is unmarshalable."
     IDS_EE_BADMARSHAL_BADMETADATA           "Invalid marshaling metadata."
     IDS_EE_BADMARSHAL_CUSTOMMARSHALER       "Custom marshalers are only allowed on classes, strings, arrays, and boxed value types."
-    IDS_EE_BADMARSHAL_GENERICS_RESTRICTION  "Generic types cannot be marshaled."
+    IDS_EE_BADMARSHAL_GENERICS_RESTRICTION  "Non-blittable generic types cannot be marshaled."
```

Comment:
If we arent changing the generics restriction everywhere, we should have both error messages and use the new one where we allow blittable generics.